sacc/tracers.py

```diff
@@ -325,3 +325,45 @@ def from_tables(cls, table_list):
                     metadata[key[5:]] = value
             tracers[name] = cls(name, z, nz, extra_columns=extra_columns, metadata=metadata)
         return tracers
+
+
+def combine_dictionaries(d1, d2, suffix1=None, suffix2=None, cmp=None):
```

Comment:
This function is not used anywhere in the PR?

Reply:
Deleted it.